tutorials/Atmos/risingbubble.jl

```diff
@@ -213,6 +215,11 @@ function config_risingbubble(FT, N, resolution, xmax, ymax, zmax)
         fast_method = LSRK144NiegemannDiehlBusch,
         timestep_ratio = 10,
     )
+    # If the user prefers a single rate explicit time integrator,
+    # the ode_solver above can be replaced with:
+    ode_solver = ClimateMachine.ExplicitSolverType(
+        solver_method = LSRK144NiegemannDiehlBusch,
+    )
```

Comment:
This overwrites the previous `ode_solver` definition -- maybe you meant to have it in a comment?

Reply:
Right - this should either be a comment in here, or point to the driver configuration setup that allows a user to choose from those options.

Reply:
I like @akshaysridhar idea

Reply:
MRRK is commented here, for the tutorial, fine with me